src/foraging_gui/Foraging.py

```diff
@@ -2699,8 +2699,8 @@ def _UpdateSuggestedWater(self,ManualWater=0):
                 # maximum 3.5ml
                 if suggested_water>3.5:
                     suggested_water=3.5
-                    self.TotalWaterWarning.setText('Supplemental water is >3.5! Health issue and LAS should be alerted!')
-                    self.TotalWaterWarning.setStyleSheet(self.default_color)
+                    self.TotalWaterWarning.setText('Supplemental water is >3.5! Health issue and \n LAS should be alerted!')
```

Comment:
Please use an if-else condition here to keep the previous text unchanged for behavior rigs (we don't need the "\n" for the behavior GUI).

Reply:
I have updated it.